src/components/AccountCreation/AccountCreation.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+// Define the type for the prop
+interface AccountCreationProps {
+  onAccountCreate: (username: string, email: string, password: string) => void;
+}
+
+const AccountCreation: React.FC<AccountCreationProps> = ({
+  onAccountCreate,
+}) => {
```

Comment:
Great job on the props here! There are a couple of different ways to present this kind of thing, including this alternate way: 

```suggestion
interface AccountCreationProps {
  onAccountCreate: (username: string, email: string, password: string) => void;
}

function AccountCreation(props:  AccountCreationProps): JSX.Element {
  const {onAccountCreate} = props;
```

In the meantime though, we said we're going to stick to vanilla JS for the new components we create for now. Can you please remove the types from these components, and add the comments at the top to disable typescript checking?

```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```